src/main/java/org/spburegistry/backend/ExceptionHandler/exception/EntityAlreadyExistsException.java

```diff
@@ -0,0 +1,9 @@
+package org.spburegistry.backend.ExceptionHandler.exception;
+
+public class EntityAlreadyExistsException extends RuntimeException {
+
+    public EntityAlreadyExistsException(String message) {
+        super(message);
```

Comment:
Why do you override superclass constructor and don't do anything else?

Reply:
This is fucking shit from spring, without constructor it doesn't work